public/index.html

```diff
@@ -8,6 +8,7 @@
     <meta name="theme-color" content="#000000" />
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
 
+    <script id="dpal" src="https://www.redhat.com/ma/dpal.js" type="text/javascript"></script>
```

Comment:
Do we need to deal with the staging url? i.e. using the other url when not on prod?

Reply:
Can we use %NODE_ENV% here to check the environment?